src/routes/authorsRoute.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const { createAuthor, findAllAuthors, findAuthorByPk, updateAuthor, deleteAuthor } = require('../controllers/authorsController');
+
+const authorsRouter = express.Router();
+
+authorsRouter.post('/', createAuthor);
+authorsRouter.get('/', findAllAuthors);
+authorsRouter.get('/:id', findAuthorByPk);
+authorsRouter.patch('/:id', updateAuthor);
+authorsRouter.delete('/:id', deleteAuthor);
```

Comment:
Theres a lot of repetition here, it can be simplified by chaining the calls together. As with the imports, its good to put one per line and indent them so its clear they belong to the opening line:
```suggestion
authorsRouter.post("/", createAuthor)
    .get("/", findAllAuthors)
    .get("/:id", findAuthorByPk)
    .patch("/:id", updateAuthor)
    .delete("/:id", deleteAuthor);
```